<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>SARS-CoV-2 Variants of Concern in Switzerland</title>
    <style>
        body {
            max-width: 1000px;
            font-family: "Nunito Sans", sans-serif;
            font-weight: 400;
            margin-left: auto;
            margin-right: auto;
        }
        .source_info {
            padding-left: 20px;
            padding-right: 20px;
            font-size: small;
        }
    </style>
</head>
<body>

<h1>SARS-CoV-2 Variants of Concern in Switzerland</h1>
<p>
    Different labs across Switzerland perform a genetic characterization of randomly chosen SARS-CoV-2 samples. All
    these samples stem from swabs of infected people. The goal is to estimate the frequency of the new variants B.1.1.7
    (described first in the UK) and 501Y.V2 (described first in South Africa). In each week, the labs report to us the
    number of considered random samples and the number of identified B.1.1.7 strains and the number of 501Y.V2 strains.
    We then employ a binomial distribution and Wilson uncertainty intervals in order to estimate the frequency of the
    variants. Given the delay between infection and swab collection, the frequency estimates for a particular week
    reflect the epidemic situation around 1-2 weeks earlier. The data is available on
    <a href="https://github.com/covid-19-Re/variantPlot/blob/master/data/data.csv">Github</a>.
</p>

<p><b>Important: the most recent estimates may be based on incomplete data and may thus be corrected later.</b></p>

<iframe
        src="https://ibz-shiny.ethz.ch/covidDashboard/variant-plot/variantPlot_b117_internal.html"
        height="500px"
        width="100%"
        style="outline: none; border: none;"
></iframe>

<iframe
        src="https://ibz-shiny.ethz.ch/covidDashboard/variant-plot/variantPlot_s501yv2_internal.html"
        height="500px"
        width="100%"
        style="outline: none; border: none;"
></iframe>


<h2>International Comparison</h2>

<iframe
        src="https://ibz-shiny.ethz.ch/covidDashboard/variant-plot/variantPlot_b117_international.html"
        height="500px"
        width="100%"
        style="outline: none; border: none;"
></iframe>

<div class="source_info">
    Sources: Denmark: <a href="https://www.covid19genomics.dk/statistics">Danish Covid-19 Genome Consortium</a>;
    UK: <a href="https://www.gisaid.org/">GISAID</a>;
    UK S-dropout: <a href="https://www.gov.uk/government/publications/investigation-of-novel-sars-cov-2-variant-variant-of-concern-20201201">
        Technical briefings of Public Health England</a>;
    Switzerland: See acknowledgements belov
</div>


<h2>Acknowledgements:</h2>

<p>
    <b>Viollier:</b> Samples are provided by <a href="https://www.viollier.ch/">Viollier AG</a>. Whole genome sequenicing is performed
    through the <a href="https://bsse.ethz.ch/cevo/research/sars-cov-2/swiss-sequencing-consortium---viollier.html">
    Swiss Viollier Sequencing Consortium</a> lead by ETH ZÃ¼rich.
</p>
<p>
    <b>Risch:</b> Samples are provided and screened by <a href="https://www.risch.ch/">Labor Risch</a>. Genomic characterization is
    performed by Labor Risch, the <a href="https://www.unispital-basel.ch/ueber-uns/bereiche/medizin/kliniken-institute-abteilungen/infektiologie-spitalhygiene/ueber-uns/abteilungen/applied-microbiology-research-laboratory/">
    University Hospital Basel (Clinical Mircobiology)</a>
    and the University Hospitals of Geneva
    (<a href="https://www.unige.ch/medecine/demed/en/groupes-de-recherche/isabella-eckerle/">Group Eckerle</a> and
    <a href="https://www.unige.ch/medecine/demed/fr/groupes-de-recherche/668kaiser/">Group Kaiser</a>).
</p>
</body>
</html>
